src/components/calcite-pick-list-item/calcite-pick-list-item.tsx

```diff
@@ -200,18 +227,36 @@ export class CalcitePickListItem {
         : selected
         ? ICONS.selected
         : ICONS.unselected;
+
     return (
       <span class={CSS.icon}>
         <calcite-icon scale="s" icon={iconName} />
       </span>
     );
   }
 
+  renderRemoveAction() {
+    if (!this.removable) {
+      return null;
+    }
+
+    return (
+      <calcite-action
+        scale="s"
+        class={CSS.remove}
+        text={this.textRemove}
+        onClick={this.removeClickHandler}
+      >
+        <calcite-icon scale="s" icon={ICONS.remove} />
```

Comment:
Now that the icon is part of the action, you can just use the icon prop on the action and remove this node.

Reply:
💥

Reply:
Thanks for the tip!